code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -629,21 +629,6 @@
 	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/abductor
 	taste_description = "something out of this world... no, universe!"
-
-/datum/reagent/mutationtoxin/android
```

Comment:
I would like to ask you to not remove this, since you can obtain it via maint pills or rarely in chapel.
The same is with IPCs. Please, it is a little part of fun in xenobiology which adds a little bit more variety into the game.

Reply:
Androids are ancient shitcode im not maintaining for maint pills.

IPC mutation toxin was greenlit for removal, It raises several questions I don't like, and breaks changelings.